plugins/modules/network_settings_workflow_manager.py

```diff
@@ -1030,6 +1023,227 @@ def get_reserve_pool_params(self, pool_info):
         self.log("Formatted reserve pool details: {0}".format(reserve_pool), "DEBUG")
         return reserve_pool
 
+    def get_dhcp_settings_for_site(self, site_id):
+        """
+        Retrieve the DHCP settings for a specified site from Cisco Catalyst Center.
+
+        Parameters:
+            self - The current object details.
+            site_id (str) - The ID of the site to retrieve DHCP settings for.
+
+        Returns:
+            dhcp_details (dict) - DHCP settings details for the specified site.
+        """
+
+        try:
+            dhcp_response = self.dnac._exec(
+                family="network_settings",
+                function='retrieve_d_h_c_p_settings_for_a_site',
+                op_modifies=True,
+                params={"id": site_id}
+            )
+            dhcp_details = dhcp_response.get("response").get("dhcp")
```

Comment:
if not dhcp_details:
            self.log("No DHCP settings found for site {0}".format(site_id), "WARNING")
            return None
print site name as well

Reply:
added the above code block and site name as well to print.